README.md

```diff
@@ -70,3 +70,16 @@ not built in `ADSupport`.
 
 Known build and runtime issues are documented in the [SwC
 wiki](http://swc.lnls.br/).
+
+## Containers
+
+### Accessing `iocsh` inside containers
+
+If a container is using the default `lnls-run` entrypoint (i.e. this won't work
```

Comment:
Shouldn't we specify close to the `docker-compose.yml` example that we have a default entrypoint that runs procServ and expects an optional argument that is a iocsh script in `RUNDIR`?

I'm worried about people not noticing that they do not need to necessarily write a entrypoint script anymore.

Reply:
Seems good to me. Thanks!